### PR TITLE
Pistons and pistons_test

### PR DESCRIPTION
**Pull request by fcastro31**
Let me know if you need me to add some function for the phase of the piston at which its moving.

### CODE REVIEW

File: .idea/Simulator2020.iml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (binitshah) [inline .idea/Simulator2020.iml]: Could you remove all `.idea/` files from this PR?

File: tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from simulator.Pistons import Pistons
```
Comment (binitshah) [inline tests/__init__.py:1]: Do you need this here?

File: tests/pistons_test.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    pistons.turn_on_piston(1)
+    assert pistons.status_list[1] == Pistons.piston_state.on
+
+    pistons.turn_off_piston(1)
+    assert pistons.status_list[1] == Pistons.piston_state.off
+
+    print(pistons.piston_status())
+
+
+test_start()
```
Comment (binitshah) [inline tests/pistons_test.py]: You can remove this line. We run these tests using `pytest tests/pistons_test.py -k "test_start"`

**Review by binitshah — CHANGES_REQUESTED** [review]
Hey, great job, I just have some small changes.

File: .idea/Simulator2020.iml (hunk 1 above)
Comment (fcastro31) [reply .idea/Simulator2020.iml]: Give me a second

File: tests/__init__.py (hunk 2 above)
Comment (fcastro31) [reply tests/__init__.py:1]: I do, remember I needed to put this last time for the test program to actually work.

File: tests/pistons_test.py (hunk 3 above)
Comment (fcastro31) [reply tests/pistons_test.py]: Got it

File: .idea/Simulator2020.iml (hunk 1 above)
Comment (fcastro31) [reply .idea/Simulator2020.iml]: Done

File: tests/pistons_test.py (hunk 3 above)
Comment (fcastro31) [reply tests/pistons_test.py]: Done

**Pull request merged**